build.gradle

```diff
@@ -101,6 +102,51 @@ task devBuild(dependsOn: [":assemble"]) {
   }
 }
 
+
```

Comment:
Would it make sense to keep this in the utilities build script?

Reply:
Since it is only used in one place it didn’t make sense to me to have it located elsewhere.